src/components/ScheduleGenerator/SchedulerPanelHeader.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <div class="requirementheader">
+    <h1 class="top header-title">Semester Schedule Builder</h1>
+    <div class="semester">
+      <button class="semester-title-button semester-title full-opacity-on-hover" :disabled="true">
+        <p class="semester-title-top">
+          <!-- Example: "Spring 2025", as on Figma. -->
+          {{ selectedSemester }}
+        </p>
+      </button>
+    </div>
+    <!-- NOTE: you cannot generate a schedule if you have not inputted any requirements. -->
+    <button class="generate-schedule-button" :disabled="generateScheduleButtonDisabled">
```

Comment:
Yep that's why I have the disabled check, if you look at the CSS then it will change color if it is not disabled.

I'm not actually sure what color it changes to, would have to examine the styles. There's a default style and then there's a style that applies when disabled is true.